### PR TITLE
fix: change file name & apply mobile view port & modify to enable pdf download & fix logo bug

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### CODE REVIEW

File: apps/web/src/common/pages/CostReportDetailPage.vue [hunk 1]
```diff
@@ -374,182 +387,178 @@ const initStatesByUrlSSOToken = async ():Promise<boolean> => {
...
+                >
+                    {{ $t('COMMON.COST_REPORT.DETAILS_BY_PRODUCT') }}
+                </p-link>
+                <p-link to="#details-by-project"
+                        class="table-link"
+                        highlight
+                >
+                    {{ $t('COMMON.COST_REPORT.DETAILS_BY_PROJECT') }}
+                </p-link>
+                <p-link to="#details-by-service-account'"
```
Comment (piggggggggy) [inline apps/web/src/common/pages/CostReportDetailPage.vue:450]: `#details-by-service-account'` 🤔

**Review by piggggggggy — APPROVED** [review]
LGTM!! 👍👍👍👍

File: apps/web/src/common/pages/CostReportDetailPage.vue [hunk 2]
```diff
@@ -350,8 +350,21 @@ const initStatesByUrlSSOToken = async ():Promise<boolean> => {
...
+const setMetaTag = () => {
+    const viewportEl = document.querySelector('head meta[name="viewport"]');
+    if (viewportEl) viewportEl.attributes.content.value = 'width=928';
+};
+const setBodyTag = () => {
+    const bodyEl = document.querySelector('body');
+    const appEl = document.querySelector('#app');
+    if (bodyEl) bodyEl.style.height = 'auto';
+    if (appEl) appEl.style.height = 'auto';
+};
```
Comment (yuda110) [inline apps/web/src/common/pages/CostReportDetailPage.vue:362]: praise: 😮

File: apps/web/src/common/pages/CostReportDetailPage.vue (hunk 1 above)
Comment (sulmoJ) [reply apps/web/src/common/pages/CostReportDetailPage.vue:450]: wow

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: apps/web/src/common/pages/CostReportDetailPage.vue (hunk 1 above)
Comment (yuda110) [reply apps/web/src/common/pages/CostReportDetailPage.vue]: wow

**Pull request merged**